emissionsapi/web.py

```diff
@@ -281,17 +291,17 @@ def get_statistics(session, interval='day', wkt=None, distance=None,
         query, limit=limit, offset=offset)
 
     return [{'value': {
-                'count': count,
-                'average': avg,
-                'standard deviation': stddev,
-                'min': min_val,
-                'max': max_val},
-            'time': {
-                'min': min_time,
-                'max': max_time,
-                'interval_start': inter}}
-            for count, avg, stddev, min_val, max_val, min_time, max_time, inter
-            in query]
+        'count': count,
```

Comment:
This seems to just make the code less readable without actually changing anything.

Reply:
[autopep8](https://pypi.org/project/autopep8/) formatted it that way. These changes slipped through. I will revert this.